x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -208,7 +208,7 @@ export function ProcessTreeNode({
                 {exitCode !== undefined && (
                   <small data-test-subj="sessionView:processTreeNodeExitCode">
                     {' '}
-                    [exit_code: {exitCode}]
+                    [exit_code: {exitCode}]{';'}
```

Comment:
why the ';' ?

Reply:
it was at the bottom before being rendered unconditionally. I thought it was supposed to be part of exit_code but it seems that it's likely a typo from before. i will remove it